activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -643,33 +643,33 @@ def build_insert_sql(insert) # :nodoc:
 
         # MySQL 8.0.19 replaces `VALUES(<expression>)` clauses with row and column alias names, see https://dev.mysql.com/worklog/task/?id=6312 .
         # then MySQL 8.0.20 deprecates the `VALUES(<expression>)` see https://dev.mysql.com/worklog/task/?id=13325 .
-        if !mariadb? && database_version >= "8.0.19"
-          values_alias = quote_table_name("#{insert.model.table_name}_values")
-          sql = +"INSERT #{insert.into} #{insert.values_list} AS #{values_alias}"
+        if supports_values_syntax?
```

Comment:
Need to update it here too.

Reply:
🤦 https://github.com/rails/rails/pull/51348/commits/2433076ac3bb768244f4c2d2c9094006f20a1f17